daisy/compute/compute.go

```diff
@@ -349,12 +382,23 @@ func (c *client) GetDisk(project, zone, name string) (*compute.Disk, error) {
 }
 
 // ListDisks gets a list of GCE Disks.
-func (c *client) ListDisks(project, zone string) (*compute.DiskList, error) {
-	d, err := c.raw.Disks.List(project, zone).Do()
-	if shouldRetryWithWait(c.hc.Transport, err, 2) {
-		return c.raw.Disks.List(project, zone).Do()
+func (c *client) ListDisks(project, zone string) ([]*compute.Disk, error) {
+	var dl []*compute.Disk
```

Comment:
Nit: other methods are using a different naming standard here. Wanna make them match?

var ds []*compute.Disk
...
for dl, err := ...